src/styles/mixins/triangle.scss

```diff
@@ -0,0 +1,52 @@
+// Size mixin
+@mixin size($width, $height: $width) {
+    width: $width;
+    height: $height;
+}
+
+// Position helper mixin
+// Article about it: http://hugogiraudel.com/2013/08/05/offsets-sass-mixin/
+@mixin _position($position, $args) {
+    @each $o in top right bottom left {
+        $i: index($args, $o);
+        @if $i
+        and $i + 1 <= length($args)
+        and type-of(nth($args, $i + 1)) == number {
+          #{$o}: nth($args, $i + 1);
+        }
+    }
+    position: $position;
+}
+
+// Absolute positioning helper mixin
+// Article about it: http://hugogiraudel.com/2013/08/05/offsets-sass-mixin/
+@mixin absolute($args) {
+    @include _position(absolute, $args);
+}
+
+// Arrow helper mixin
+@mixin triangle($direction, $position, $color: white, $size: 1rem) {
+  // Make sure the direction is valid
+    @if not index(top right bottom left, $direction) {
+      @warn 'Direction must be one of top, right, bottom or left.';
+    } @else {
+      @include absolute($position);
+      @include size(0);
+      content: '';
+      z-index: 2;
+
+      border-#{$direction}: $size solid $color;
+      $perpendicular-borders: $size solid transparent;
+
+      @if $direction == top or $direction == bottom {
+        border-left: $perpendicular-borders;
+        border-right: $perpendicular-borders;
+        transform: translateX(-50%);
+
+      } @else if $direction == right or $direction == left {
+        border-bottom: $perpendicular-borders;
+        border-top: $perpendicular-borders;
+        transform: translateY(-50%);
+      }
+    }
+}
```

Comment:
Is this file part of the nav header changes, or was it pulled in from a merge with Maher's branch at some point?  This is a good example of why it's better to rebase, not merge!